profiles/mpl_funcs.py

```diff
@@ -0,0 +1,108 @@
+from matplotlib.offsetbox import AnnotationBbox, OffsetImage
+from matplotlib.animation import FuncAnimation
+
+from main_window.competition.comp_app import MAP_MARKER
+from main_window.device_manager import DeviceType
+from main_window.data_entry_id import DataEntryIds
+from main_window.main_app import MainApp
+from textwrap import wrap
+
+from main_window.mplwidget import MplWidget
+
+label_to_dataID = {"Altitude":DataEntryIds.CALCULATED_ALTITUDE,
```

Comment:
Can we also include labels for stage 2?

Reply:
<img width="980" alt="Screen Shot 2022-05-18 at 8 28 56 PM" src="https://user-images.githubusercontent.com/77385101/169198251-c076dcc3-abd1-4cb0-a678-8b87448d5796.png">
Good idea, added stage 2 :)

The mappings are basically the same for every rocket, so didn't think having one for each profile would help much.